test/Tabs.test.js

```diff
@@ -136,6 +136,7 @@ describe('tabs behaviour', () => {
 			fixtures.reset();
 			fixtures.insertSimple();
 			withUpdateUrl && tabsEl.setAttribute('data-o-tabs-update-url', '');
+			tabsEl.setAttribute('data-o-tabs-disablefocus', 'false');
```

Comment:
Yeh I thought that but the test was failing... The test is now failing for a random script error so when that's sorted I'll try to test this out properly again.